clientv3/retry_interceptor.go

```diff
@@ -48,11 +48,21 @@ func (c *Client) unaryClientInterceptor(logger *zap.Logger, optFuncs ...retryOpt
 			if err := waitRetryBackoff(ctx, attempt, callOpts); err != nil {
 				return err
 			}
+			logger.Info(
+				"retrying unary intercept",
+				zap.String("target", cc.Target()),
+				zap.Uint("attempt", attempt),
+			)
 			lastErr = invoker(ctx, method, req, reply, cc, grpcOpts...)
-			logger.Info("retry unary intercept", zap.Uint("attempt", attempt), zap.Error(lastErr))
 			if lastErr == nil {
 				return nil
 			}
+			logger.Warn(
+				"retrying unary intercept failed",
```

Comment:
I understand this function is an interceptor, but the actual function call we are retrying is the invoker.  So this message (and other similar messages in this function) sounds a little wired to me.  It sounds like some interception failed, which is not what actually happened.  Maybe something like "retrying of unary invoker failed" or simply "retrying failed"?

Reply:
Good point. Updated. PTAL.

Reply:
lgtm